library/std/src/keyword_docs.rs

```diff
@@ -16,17 +16,16 @@
 ///
 /// In general, any cast that can be performed via ascribing the type can also be done using `as`,
 /// so instead of writing `let x: u32 = 123`, you can write `let x = 123 as u32` (Note: `let x: u32
-/// = 123` would be best in that situation). The same is not true in the other direction, however,
-/// explicitly using `as` allows a few more coercions that aren't allowed implicitly, such as
-/// changing the type of a raw pointer or turning closures into raw pointers.
+/// = 123` or `let x = 123u32` would be best in that situation). The same is not true in the other
```

Comment:
Which way is preferred: `let x: u32 = 123` or `let x = 123u32`? Or I can just remove that addition if it would require too much bikeshedding :)

Reply:
I don't like `123u32`, either `let x: u32` or `let x = 123_u32` would be my preference. Probably the former.

Reply:
I ended up just getting rid of the `let x = 123u32` example.